docs/contributing/contributing-rules.md

```diff
@@ -39,24 +39,9 @@ See an example of a [pull request](https://github.com/returntocorp/semgrep-rules
 
 Find more about the rules repository by reading the [Rule writing](#rule-writing) and [Tests](#tests) sections.
 
-#### Installing pre-commit
+#### Installing `pre-commit`
 
-Install pre-commit if you are contributing to the rules repository through GitHub but you are **not** contributing through sharing your rule in Semgrep App.
-
-1. Install [pre-commit](https://pre-commit.com/) by issuing the following command:
-    ```sh
-    python -m pip install pre-commit
-    ```
-2. Install the pre-commit hooks:
-    ```sh
-    pre-commit install
-    ```
-To check if `pre-commit` is working as expected, run the following command:
-    ```sh
-    pre-commit run --all
-    ```
-
-Once `pre-commit` is set up you may commit code and create pull requests.
+If you are contributing to the rules repository through GitHub but you are **not** contributing through sharing your rule in Semgrep App, install `pre-commit`. Follow [pre-commit installation documentation](https://pre-commit.com/#installation). Once `pre-commit` is set up you may commit code and create pull requests to rules repository.
```

Comment:
@minusworld I would normally agree but `pre-commit` is not our tool, we can provide a link to their docs. We should not document how to use it unless we have a very specific way in which it is used, because it can mean that we have to maintain docs of other companies...